docs/transports.md

```diff
@@ -525,6 +526,33 @@ logger.info('Will NOT appear')
 logger.info({ [enable]: false }, 'Will appear immedialty')
 logger.info('Will NOT appear')
 ```
+<a id="pino-openobserve"></a>
+### @openobserve/pino-openobserve
+
+[@openobserve/pino-openobserve](https://github.com/openobserve/pino-openobserve) is a Pino v7+ transport that can send the logs to [OpenObserve](https://openobserve.ai) Instance.
+
+```
+import pino from 'pino';
+import OpenobserveTransport from '@openobserve/pino-openobserve';
+
+const logger = pino({
+  level: 'info',
+  transport: {
+    target: OpenobserveTransport,
+    options: {
+      url: 'https://your-openobserve-server.com',
+      organization: 'your-organization',
+      streamName: 'your-stream',
+      auth: {
+        username: 'your-username',
+        password: 'your-password',
+      },
+    },
+  },
+});
+```
```

Comment:
The other examples in this document are CJS. Please update yours to be consistent with the others.

Reply:
This example is updated with the CJS